src/bindings/cpp/include/keyset.hpp

```diff
@@ -31,7 +31,9 @@ class KeySet
 	inline KeySet(ckdb::KeySet *k);
 	inline KeySet(const KeySet &other);
 
-	inline explicit KeySet(size_t alloc, va_list ap);
+#if defined(__amd64__) || defined(__x86_64__) || defined(__i386__)
```

Comment:
obviously thats not exactly the list that do not have va_list defined in a way that its not a pointer

actually, maybe its better to only provide fromVA then